lib/gtfs/csv.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Gtfs.Csv do
+  @moduledoc """
+  Manage parsing data from GTFS CSV files.
+  """
+
+  @type row :: %{required(String.t()) => String.t()}
+
+  @doc """
+  Takes binary csv data, a function to filter each row, and a function to parse each row, and returns the list of results.
+  The rows will be passed to the parser as maps with string keys and values.
+  e.g. %{"col1" => "1", "col2" => "x"}
+
+  iex> Gtfs.Csv.parse("col1,col2\\n1,x\\n2,y\\n3,z", fn row -> row["col2"] != "y" end, fn row -> String.to_integer(row["col1"]) end)
+  [1, 3]
+
+  exposed for testing
```

Comment:
This comment is now out of date.

Reply:
I just mean the `exposed for testing` line. It's now exposed for regular use in `gtfs.ex`.

Reply:
Oh right, very true!